src/app/modules/item/http-services/get-item-parents.service.ts

```diff
@@ -0,0 +1,76 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { appConfig } from 'src/app/shared/helpers/config';
+import { ItemPermissionsInfo } from '../helpers/item-permissions';
+
+interface RawItemParent extends ItemPermissionsInfo {
+  id: string,
+  best_score: number,
+  string: {
+    title: string|null,
+  },
+  category: 'Undefined'|'Discovery'|'Application'|'Validation'|'Challenge',
+  type: 'Chapter'|'Task'|'Course'|'Skill',
+  result: {
+    attempt_id: string,
+    latest_activity_at: string,
+    started_at: string|null,
+    score_computed: number,
+    validated: boolean,
+  },
+}
+
+export interface ItemParent extends ItemPermissionsInfo {
+  id: string,
+  bestScore: number,
+  string: {
+    title: string|null,
+  },
+  category: 'Undefined'|'Discovery'|'Application'|'Validation'|'Challenge',
+  type: 'Chapter'|'Task'|'Course'|'Skill',
+  result: {
+    attempt_id: string,
+    latestActivityAt: Date,
+    startedAt: Date|null,
+    score: number,
+    validated: boolean,
+  },
+}
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GetItemParentsService {
+
+  constructor(private http: HttpClient) { }
+
+  get(id: string, attemptId: string): Observable<ItemParent[]> {
+    // eslint-disable-next-line no-console
+    console.log('call');
```

Comment:
!!!
... but why the hell have you put this `eslint-disable`?  It is there especially so that you do not forget to remove the console.log at the end.